### ISSUE TITLE
Program assertion fail when creating new PhysicsScene

### ISSUE THREAD

**Issue by rw0xnan** [FlaxEngine/FlaxEngine#2180]
Program assertion fail when creating new PhysicsScene in Version 1.7.6407.2

**Issue description:**
Creating a new physics scene in C# with the following expression:
```
Physics.FindOrCreateScene("Test");
```
causes the program to exit with the following error:
```
File: F:\FlaxEngine\Source\Engine\Physics\Physics.cpp
Line: 462

Expression: IsInMainThread() && !_isDuringSimulation 
```

The program was expected to create another physics scene that can be switched between during runtime. I was running the PhysicsWorld.zip program to preview the feature, which can be found here: https://github.com/FlaxEngine/FlaxEngine/pull/673

**Steps to reproduce:**
Create a C# script with ```Physics.FindOrCreateScene("Test");```
Attach to actor
Play

**Minimal reproduction project:**
[MyProject.zip](https://github.com/FlaxEngine/FlaxEngine/files/13931636/MyProject.zip)

**Flax version:**
Version 1.7.6407.2

**Comment by icecold328** [comment]
i am also having the same problem with the same version of flax.

**Comment by mafiesto4** [comment]
Fixed in 6d757946c5c887649f50fdcf7f7f60b9abe7cdef.
